ci-operator/jobs/openshift/installer/openshift-installer-master-presubmits.yaml

```diff
@@ -2772,6 +2772,72 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )images,?($|\s.*)
+  - agent: kubernetes
+    always_run: false
+    branches:
+    - master
+    cluster: build01
+    context: ci/prow/openstack-install-multiplenetworks
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      ci-operator.openshift.io/prowgen-controlled: "true"
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-openshift-installer-master-openstack-install-multiplenetworks
```

Comment:
If you don’t want to block merges, you need “optional: true” here (otherwise, the test becomes voting as soon as you request it with ‘/test openstack-install-multiplenetworks”)